tasks.py

```diff
@@ -239,3 +239,34 @@ def mv_usage_stats_json(ctx):
     cmd = cmd.format(outfile)
     ctx.run(cmd)
     print(f"'{outfile}' copied to dbfs.")
+
+
+@invoke.task
+def docker(ctx, name="gx38", tag="dev", source=pathlib.Path.cwd(), build=False):
```

Comment:
Both the docker build and run commands both make the assumption that you are in the root repo directory so I think we can remove `source` as a parameter for now and set it to the root directly. I think it might also be confusing to have it as a parameter currently since we have the docker build-arg source.

Doing that and adding a docstring already adds value to what we have now. 

I could see us wanting to parameterize the command and have it default to `bash`.

Thanks for getting this started! I'm happy to pick this up and make the edits I suggest if you want to step back from this to focus on your thin-saas work. If you want to continue with this, that is also good with me.

Thanks!